openpype/settings/defaults/project_settings/maya.json

```diff
@@ -1,5 +1,5 @@
 {
-    "open_workfile_post_initialization": false,
+    "open_workfile_post_initialization": true,
```

Comment:
Why are we making this the default?

Reply:
This is solving issues with maya plugin init when opening some scenes and it doesn't introduce any additional side effects, so I guess it is better to have it on by default.

Reply:
It was only intended for XGEN right?